### ISSUE TITLE
Issue with docs/tasks/traffic-management/egress/egress-gateway-tls-origination/index.md

### ISSUE THREAD

**Issue by woodliu** [istio/istio.io#7512]
I follow the [steps](https://istio.io/docs/tasks/traffic-management/egress/egress-gateway-tls-origination/#redeploy-the-egress-gateway-with-the-client-certificates) to learn istio. But when i finish the steps, the egress become STALE：
```
# istioctl ps
NAME                                                   CDS        LDS                            EDS        RDS          PILOT                       VERSION
istio-egressgateway-7649c49d8-jqtq6.istio-system       SYNCED     STALE (Never Acknowledged)     SYNCED     NOT SENT     istiod-788cf6c878-chfnk     1.6.0
istio-ingressgateway-569669bb67-x59j5.istio-system     SYNCED     SYNCED                         SYNCED     NOT SENT     istiod-788cf6c878-chfnk     1.6.0
prometheus-79878ff5fd-n9d57.istio-system               SYNCED     SYNCED                         SYNCED     SYNCED       istiod-788cf6c878-chfnk     1.6.0
sleep-78484c89dd-2vvvh.default                         SYNCED     SYNCED                         SYNCED     SYNCED       istiod-788cf6c878-chfnk     1.6.0

```
The istiod's log shows：
```
2020-06-09T06:40:48.921913Z     warn    ads     ADS:LDS: ACK ERROR router~10.80.3.112~istio-egressgateway-7649c49d8-9g6lr.istio-system~istio-system.svc.cluster.local-201 Internal:Error adding/updating listener(s) 0.0.0.0_443: Invalid path: /etc/certs/root-cert.pem

2020-06-09T06:40:48.937023Z     warn    ads     ADS:LDS: ACK ERROR router~10.83.0.86~istio-egressgateway-7649c49d8-jqtq6.istio-system~istio-system.svc.cluster.local-200 Internal:Error adding/updating listener(s) 0.0.0.0_443: Invalid path: /etc/certs/root-cert.pem
```
I check the path `/etc/certs/` in egress pod, but can't find it. Where is the path and how it been generated?

And when i clean the things deployed in this task, the egress become normal again:
```
# istioctl ps
NAME                                                   CDS        LDS        EDS        RDS          PILOT                       VERSION
istio-egressgateway-7649c49d8-jqtq6.istio-system       SYNCED     SYNCED     SYNCED     NOT SENT     istiod-788cf6c878-chfnk     1.6.0
istio-ingressgateway-569669bb67-x59j5.istio-system     SYNCED     SYNCED     SYNCED     NOT SENT     istiod-788cf6c878-chfnk     1.6.0
prometheus-79878ff5fd-n9d57.istio-system               SYNCED     SYNCED     SYNCED     SYNCED       istiod-788cf6c878-chfnk     1.6.0
```

**Comment by woodliu** [comment]
Does anyone help？

**Comment by GregHanson** [comment]
Related issue https://github.com/istio/istio/issues/23910

**Comment by JimmyCYJ** [comment]
cc @howardjohn do you have any idea what would cause Istiod generate that log "Invalid path: /etc/certs/root-cert.pem"?

**Comment by howardjohn** [comment]
Sounds like they have a Gateway or something that is invalid. https://istio.io/latest/docs/tasks/traffic-management/egress/egress-gateway-tls-origination/#configure-mutual-tls-origination-for-egress-traffic the doc is wrong, needs to use SDS

**Comment by JimmyCYJ** [comment]
Tentatively assign to @GregHanson as this looks similar to https://github.com/istio/istio/issues/23910.
Also cc @incfly

**Comment by GregHanson** [comment]
Fixes merged and back ported to 1.6 release.  Docs have been updated.  Will be reflected in next releases

**Comment by incfly** [comment]
@JimmyCYJ FYI the fix should be covered in the 1.7 release notes.
